### DISCUSSION TITLE
My cluster keep recreating new resources

### DISCUSSION THREAD

**Discussion by krasimirdermendzhiev** [mysticaltech/terraform-hcloud-kube-hetzner#586] [Q&A]
Hello, all
I need help for my cluster.
I have one question why my resources:
```
cluster-autoscaler 
coredns
hcloud-cloud-controller-manager
hcloud-csi-controlle
```
keep recreating and and my nodes is restarting
```
hcloud-csi-node-1
hcloud-csi-node-2
hcloud-csi-node-3
```

**Comment by mysticaltech** [comment]
@krasimirdermendzhiev It's the node's automatic updates. Make sure you run in HA (see readme and kube.tf), otherwise turn the upgrade off (see readme). Hope this helps!
